app/src/debug/java/org/simple/clinic/DebugClinic.kt

```diff
@@ -0,0 +1,35 @@
+package org.simple.clinic
+
+import android.annotation.SuppressLint
+import com.facebook.stetho.Stetho
+import com.tspoon.traceur.Traceur
+import org.simple.clinic.di.AppComponent
+import org.simple.clinic.di.AppModule
+import org.simple.clinic.di.DaggerAppComponent
+import org.simple.clinic.sync.SyncScheduler
+import timber.log.Timber
+import javax.inject.Inject
+
+@SuppressLint("Registered")
+class DebugClinic : Clinic() {
+
+  @Inject
+  lateinit var syncScheduler: SyncScheduler
+
+  override fun onCreate() {
+    super.onCreate()
+
+      if (BuildConfig.DEBUG) {
```

Comment:
Since `DebugClinic` class only exists in the debug source set, this condition will always be true :)

Reply:
Yeah please remove this condition.